src/include/ikarus/finiteElements/mechanics/enhancedAssumedStrains.hh

```diff
@@ -1,25 +1,6 @@
-
-// /*
-//  *  This file is part of the Ikarus distribution (https://github.com/rath3t/Ikarus).
-//  *  Copyright (c) 2021 Alexander Müller.
-//  *  Institut fuer Baustatik und Baudynamik
-//  *  Universität Stuttgart
-//  *
-//  *  This library is free software; you can redistribute it and/or
-//  *   modify it under the terms of the GNU Lesser General Public
-//  *   License as published by the Free Software Foundation; either
-//  *   version 2.1 of the License, or (at your option) any later version.
-//
-// *   This library is distributed in the hope that it will be useful,
-// *   but WITHOUT ANY WARRANTY; without even the implied warranty of
-// *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
-// *   Lesser General Public License for more details.
-//
-// *   You should have received a copy of the GNU Lesser General Public
-// *   License along with this library; if not, write to the Free Software
-// *   Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301
-// *  USA
-// *
+// Copyright (c) 2021 Alexander Müller.
```

Comment:
This line isn't consistent in all files? It is missing in some and in some it says IkarusDevelopers?

Reply:
This should be consistent did you find more than these two?

Reply:
No I only found in these two. It was missing in the rest.

Reply:
done